Sources/Public/CalendarViewRepresentable.swift

```diff
@@ -65,11 +65,16 @@ public struct CalendarViewRepresentable: UIViewRepresentable {
 
   public func makeUIView(context: Context) -> CalendarView {
     let calendarView = CalendarView(initialContent: makeContent())
+    calendarView.backgroundColor = .clear
+    calendarView.directionalLayoutMargins = .zero
     proxy?._calendarView = calendarView
     return calendarView
   }
 
   public func updateUIView(_ calendarView: CalendarView, context: Context) {
+    calendarView.backgroundColor = backgroundColor ?? calendarView.backgroundColor
+    calendarView.directionalLayoutMargins = layoutMargins ?? calendarView.directionalLayoutMargins
```

Comment:
Question, this affects the margins and background on the days of week header?

Reply:
It would be nice to set a background just for the header.. what do you think?

Reply:
Oh yikes, this is a great callout! Let me think through that API a bit...